BootloaderCommonPkg/Include/Guid/DeviceTableHobGuid.h

```diff
@@ -0,0 +1,47 @@
+/** @file
+  This file defines the hob structure for the device table.
+
+  Copyright (c) 2018, Intel Corporation. All rights reserved.<BR>
+  This program and the accompanying materials
+  are licensed and made available under the terms and conditions of the BSD License
+  which accompanies this distribution.  The full text of the license may be found at
+  http://opensource.org/licenses/bsd-license.php.
+
+  THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+  WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
+
+**/
+
+#ifndef __DEVICE_TABLE_HOB_GUID_H__
+#define __DEVICE_TABLE_HOB_GUID_H__
+
+///
+/// Device table Hob GUID
+///
+extern EFI_GUID gDeviceTableHobGuid;
+
```

Comment:
#pragma pack(1)

Reply:
It is not required since the structure is 4 bytes aligned in purpose. and it is the preferred to avoid to use pack()